packages/ptokens-assets-evm/test/utils/viem-clients.ts

```diff
@@ -0,0 +1,13 @@
+import { PublicClient, Transport, WalletClient, createPublicClient, createWalletClient, http } from 'viem'
+import { arbitrum } from 'viem/chains'
+
+export const walletClient = createWalletClient({
```

Comment:
Are you using a real node?
You should mock it. Tests should work without an internet connection.
Try to go airplane mode and test. If it fails you have work to do.